EIPS/eip-1872.md

```diff
@@ -0,0 +1,171 @@
+
+### Priority Network Upgrades
+
+Priority network upgrades are reserved for upgrades that require more urgency
+than a roadmap network upgrade yet do not present a systemic risk to the network
+or the ecosystem. To date there have been no examples of a priority upgrade.
+Possible examples may include roadmap upgrades that need to occur in multiple
+upgrades or for security risks that have a existing mitigation in place that
+would be better served by a network upgrade. Another possible reason may be to
+defuse the difficulty bomb due to postponed roadmap upgrades.
+
+Priority network upgrades are best launched in unused roadmap launch windows,
+namely the third week of January, April, July, and October. If necessary they
+may be launched in the third week of any month, but strong consideration and
+preference should be given to unused roadmap launch windows.
+
+Priority network upgrades should be announced and a block chosen far enough in
+advance so major clients implementors can release software with the needed block
+number in a timely fashion. It is expected that two to four weeks would be
+sufficient.
+
+### Critical Network Upgrades
+
+Critical network upgrades are network upgrades that are designed to address
+systemic risks to the protocol or to the ecosystem. Historical examples include
+Dao Fork, Tangerine Whistle, and Spurious Dragon.
+
+This EIP provides neither guidance nor restrictions to the development and
+deployment of these emergency hard forks. These upgrades are typically launched
+promptly after a solution to the systemic risk is agreed upon between the client
+implementors.
+
+### Network Upgrade Naming
+
+The choice of a name for a network upgrade can help communicate what type of
+network upgrade it is. There have been patterns used in the past that if
+continued would help communicate this.
+
+Roadmap Network Upgrades have historically been given single word names. The
+first three network upgrades were Frontier, Homestead, and Metropolis.
+Metropolis was split into two upgrades, Byzantium and Constantinople. As of the
+writing of this EIP the next network upgrade will be called Istanbul.
+
+Critical Network Upgrades have historically been two word phrases. Sometimes the
+name applies to the issue at hand, sometimes it do not. Previous Critical
+Network upgrade names have been Dao Fork, Tangerine Whistle, Spurious Monkey,
+and Constantinople Fix.
+
+### Network Upgrade Block Number Choice
+
+When choosing an activation block the number can be used to communicate the role
+of a particular network in the Ethereum Ecosystem.
+
+To date all Mainnet activation blocks have ended in three or more zeros,
+including Critical Network Upgrades. Ropsten and Kovan initially started with
+three zeros but switched to palindromic numbers. Rinkeby has always had
+palindromic activation blocks. Goerli has yet to perform a network upgrade.
+
+To continue this pattern network upgrade activation block numbers for mainnet
```

Comment:
I think this section should strive to pick a block number "mid-week" for upgrades, where mid-week is between Tuesday to Thursday UTC, as that should maximize the amount of resources available from all teams if issue with the fork occur.  IMO this is even more important with the testnets, where most of the "rocky" forks have occurred (either due to lack of miners for the new PoW chains or consensus issues around PoA implementations).

Reply:
I said Wednesday noon UTC+0 to maximize the likely hood of a monkey fighting fork on a monday through friday plane.